packages/devtools-reps/src/launchpad/constants.js

```diff
@@ -4,6 +4,7 @@
 
 module.exports = {
   ADD_EXPRESSION: Symbol("ADD_EXPRESSION"),
+  LOAD_OBJECT: Symbol("LOAD_OBJECT"),
   EVALUATE_EXPRESSION: Symbol("EVALUATE_EXPRESSION"),
   CLEAR_EXPRESSIONS: Symbol("CLEAR_EXPRESSIONS"),
   SHOW_RESULT_PACKET: Symbol("SHOW_RESULT_PACKET"),
```

Comment:
Hmm, we've switched to using plain strings in the debugger because Flow does the validation for us.